x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared/data_tier_allocation_field/components/data_tier_allocation.scss

```diff
@@ -0,0 +1,9 @@
+.indexLifecycleManagement__phase__dataTierAllocation {
+  &__controlSection {
```

Comment:
You mentioned this is a copy/paste of a different file. I see there are now two files named the same and with the exact same selector and styles. Is there a reason not to reuse the component instead of duplicating?

<img width="1266" alt="Screen Shot 2020-10-22 at 15 30 18 PM" src="https://user-images.githubusercontent.com/549577/96920913-cf840200-147b-11eb-9c49-365b8c55edd1.png">

Reply:
Hi @cchaos ! Thanks for the review, this file will be removed in a subsequent PR which will be started directly after this PR. This work is part of enabling a bigger redesign for ILM and it is easier to manage and review the new code in stages.

Reply:
👍 Gotcha. Then the only other suggestion I have is to use BEM naming with a 3-letter feature prefix like we do in other solutions. That would turn this selector into `ilmDataTierAllocation__controlSection` for simplicity.